src/v2/Apps/Order/Components/TransactionDetailsSummaryItem.tsx

```diff
@@ -52,6 +54,33 @@ export class TransactionDetailsSummaryItem extends React.Component<
         <Spacer mb={2} />
         <Entry label="Total" value={this.buyerTotalDisplayAmount()} final />
         {showOfferNote && order.mode === "OFFER" && this.renderNoteEntry()}
+        {order.state === "SUBMITTED" && (
```

Comment:
Late to this party. But wondering if that is the correct state to be looking at. The Submitted state does not necessarily guarantee you the purchase and we def have a lot of orders and even more offers that will get rejected by galleries. Logically this collection item creation should happen on order Approval I think rather than on order submission.

Please ignore if that point was discussed and decided upon already :)

Reply:
@oxaudo Very good catch. I'm not sure how quickly the order status can change from `Submitted` to `Approved` as the gallery could also approve the order very late. But the requirement is to display this message when the user clicks `Submit` which is on the BNMO page. If we are showing it based on the approval status, then there is no point in viewing it on this page.

I will post about this in the CX channel and ask PM and designer. Thanks for noticing it 🙌